packages/twenty-server/src/engine/metadata-modules/remote-server/dtos/update-remote-server.input.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+import { IsOptional } from 'class-validator';
+import GraphQLJSON from 'graphql-type-json';
+
+import {
+  ForeignDataWrapperOptions,
+  RemoteServerType,
+  UserMappingOptions,
+} from 'src/engine/metadata-modules/remote-server/remote-server.entity';
+import { UserMappingOptionsInput } from 'src/engine/metadata-modules/remote-server/utils/user-mapping-options-input.utils';
+
+@InputType()
+export class UpdateRemoteServerInput<T extends RemoteServerType> {
+  @Field(() => String)
+  id: T;
```

Comment:
why is id a server type?

Reply:
its a mistake thanks for spotting this !